### DISCUSSION TITLE
Move assets from S3 back to local filesystem

### DISCUSSION THREAD

**Discussion by iannuzzelli** [mastodon/mastodon#18844] [Q&A]
Hello all, I have been self-hosting a Mastodon community for some time on AWS, relying on S3 for media assets and cache, served by a cloudfront distribution with a dedicated hostname. All dockerised, all working well, but I have now to migrate the whole thing to Azure.
And while it has been relatively straightforward to move all the containers, I now face the problem that Paperclip does not support Azure storage (see #14810). I guess I have to serve all assets from the local filesystem.
I tried to sync back all assets from S3 to the local volume (under `public/system`), but they are not exposed...
Any suggestions? thanks!

**Comment by tmkis2** [comment]
Sadly no answer here - have the same question.....
